qemu/tests/softlockup.py

```diff
@@ -38,8 +38,7 @@ def run(test, params, env):
 
     vm = env.get_vm(params["main_vm"])
     login_timeout = int(params.get("login_timeout", 360))
-    auto_dir = os.environ.get("AUTODIR", os.environ.get("AUTOTEST_PATH"))
-    stress_dir = os.path.join(auto_dir, "tests", "stress")
+    stress_dir = data_dir.get_deps_dir("stress")
```

Comment:
I guest, here should be   os.path.join(data_dir.get_data_dir(), 'stress') ?

Reply:
@xutian "get_deps_dir" is ok here, it will try to find "stress" in outermost layer "deps".
In short,  from "qemu.tests.softlockup" , it will try to find "stress" from "qemu.tests.deps", if not, then "qemu.deps", and then "deps".